phpBB/styles/prosilver/theme/colours.css

```diff
@@ -1163,6 +1163,30 @@ input.disabled {
 	border-color: #dcdcdc;
 }
 
+/* smiley dropbox
+ ----------------------------------------- */
+.button .dropdown-extended .pointer-inner {
+	border-color: #ffffff transparent;
+}
+
+.dropdown .smiley-content {
+	background: #ffffff;
+	border-color: #b9b9b9;
+	box-shadow: 1px 3px 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown-up .smiley-content {
+	box-shadow: 1px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.smiley-box .smiley-list {
+	background-color: #ffffff;
+}
+
+.smiley-box .smiley-list a {
+	background-color: #ffffff;
+}
+
 /* notifications */
```

Comment:
the only change should be to restyle the individual li using the new class suggested